extensions/notebook/package.nls.json

```diff
@@ -2,7 +2,9 @@
 	"displayName": "Notebook Core Extensions",
 	"description": "Defines the Data-procotol based Notebook contribution and many Notebook commands and contributions.",
 	"notebook.configuration.title": "Notebook configuration",
+	"notebook.python.configuration.title": "Notebook python configuration",
 	"notebook.enabled.description": "Enable viewing notebook files using built-in notebook editor.",
+	"notebook.pythonPath.description": "Python installation path on local machine used by Notebook.",
```

Comment:
"Local path to Python installation used by Notebooks"?

Reply:
Seems good. Changed it.